journal-club-paper/intro.in

```diff
@@ -1,130 +1,55 @@
+to increase patient comfort by reducing the need for multiple attempts.
+Intra-operative imaging can assist with this, but it does not account
+for unmodeled needle deflection or target shift. This can be compensated
+for by rotating a bevel-tipped needle according to a closed-loop image-guided
+control system. A fully autonomous needle insertion system is not desirable
+as it typically removes the physician entirely from patient contact, so
+a cooperative control system is presented. The physician applies an
+input force directly while the active compensation through needle rotation
+is performed autonomously.
+
+This work was motivated by the robotics system described in Eslami
+\textit{et al}\textsuperscript{REFERENCE 7 HERE}. While this system
+aligned the needle correctly, it still suffered from positional
+inaccuracy introduced by unmodeled needle deflection and target shift.
+The goal of this research is to implement a cooperative controlled
+needle correction system with closed-loop active compensation. This
+system must be compatible with an MR environment and have needle
+localization in real-time MR images as shown in Patel \textit{et al}\textsuperscript{REFERENCE 18 here}
+
+The error in needle accuracy due to deflection, and the amount
+of active compensation possible, is limited by the mechanical 
+properties of the needle. Prior work to model this has been widely implemented
+using the kinematic bicycle model\textsuperscript{REFERENCE 32 here}
+as well as a joint kinematic and dynamic system.\textsuperscript{REFERENCE 24 here}
+Other approaches (REFERNCE 6, 14) use a duty-cycled approach for 
+needle steering. This approach requires insertion and rotation to
+be tightly coupled, prohibiting cooperative control. Rossa \textit{et al},
+\textsuperscript{reference 21} presented a similar system with manual
+insertion and autonomous rotation, however it did not have a
+cooperative insertion approach, which increases the accuracy of the
+insertion axis as well.
+
+A cooperative approach was chosen in this system to maintain
+phsyician interaction with the patient. Fully autonomous, isolated
+solutions are more likely to face increased regulatory scrutiny.\textsuperscript{REFERENCE 20}
+Teleoperated systems have been developed to address this issue(REFERENCE 28, 23, 17).
+These systems often employ haptic or vibrational feedback to assist the
+physician in positioning the needle, however they still isolate the
+physician physically from the patient. A cooperative control scheme
+has the benefits of teleoperation while keeping the physician in the
+same room as the patient. There are many successful cooperative devices
+in use today(REFERENCE 22, 11, 3, 13).
+
+Prior work in steering needles via needle deflection has focused
+on thin, flexible needles which are not suitable for a clinical
+biopsy scenario. The approach presented here is focused on a
+clinical translation.\textit{The primary contributions of this
```

Comment:
Since the whole italics part is pretty much a direct quote from the original paper, we probably want to add the original paper as a reference here. In the rubric for the assignment he seems to give weight for appropriately citing the original paper.